src/test/resources/Observation/create-pac02.json

```diff
@@ -0,0 +1,65 @@
+{
+  "resourceType": "Observation",
+  "id": "6a78560b-2f7c-4af5-a7bc-3c7d62ca8458",
+  "meta": {
+    "profile": [
+      "https://www.netzwerk-universitaetsmedizin.de/fhir/StructureDefinition/carbon-dioxide-partial-pressure"
+    ]
+  },
+  "identifier": [
+    {
+      "type": {
+        "coding": [
+          {
+            "system": "http://terminology.hl7.org/CodeSystem/v2-0203",
+            "code": "OBI"
+          }
+        ]
+      },
+      "system": "https://www.charite.de/fhir/CodeSystem/lab-identifiers",
+      "value": "2019-8_paCO2",
+      "assigner": {
+        "reference": "Organization/Charité"
+      }
+    }
+  ],
+  "status": "final",
+  "category": [
+    {
+      "coding": [
+        {
+          "system": "http://loinc.org",
+          "code": "26436-6"
+        },
+        {
+          "system": "http://terminology.hl7.org/CodeSystem/observation-category",
+          "code": "laboratory"
+        },
+        {
+          "system": "http://loinc.org",
+          "code": "18767-4"
+        }
+      ]
+    }
+  ],
+  "code": {
+    "coding": [
+      {
+        "system": "http://loinc.org",
+        "code": "2019-8",
+        "display": "Carbon dioxide [Partial pressure] in Arterial blood"
+      }
+    ],
+    "text": "Carbon dioxide [Partial pressure] in Arterial blood"
+  },
+  "subject": {
+    "reference": "Patient/c786ac3c-0579-4aa2-adda-db784b214ad6"

```

Comment:
Subject is not correct needs to be fixed

Reply:
Adjusted to:
```
"subject": {
    "identifier": {
      "system": "urn:ietf:rfc:4122",
	  "value": "{{patientId}}"
	}
  }
```